test/functional/services/combo_box.ts

```diff
@@ -112,28 +132,42 @@ export function ComboBoxProvider({ getService, getPageObjects }) {
       return optionsText;
     }
 
-    async doesComboBoxHaveSelectedOptions(comboBoxSelector) {
+    public async doesComboBoxHaveSelectedOptions(comboBoxSelector: string): Promise<boolean> {
       log.debug(`comboBox.doesComboBoxHaveSelectedOptions, comboBoxSelector: ${comboBoxSelector}`);
       const comboBox = await testSubjects.find(comboBoxSelector);
-      const selectedOptions = await comboBox.findAllByClassName('euiComboBoxPill', WAIT_FOR_EXISTS_TIME);
+      const selectedOptions = await comboBox.findAllByClassName(
+        'euiComboBoxPill',
+        WAIT_FOR_EXISTS_TIME
+      );
       return selectedOptions.length > 0;
     }
 
-    async getComboBoxSelectedOptions(comboBoxSelector) {
+    public async getComboBoxSelectedOptions(comboBoxSelector: string): Promise<string[]> {
       log.debug(`comboBox.getComboBoxSelectedOptions, comboBoxSelector: ${comboBoxSelector}`);
       return await retry.try(async () => {
         const comboBox = await testSubjects.find(comboBoxSelector);
-        const selectedOptions = await comboBox.findAllByClassName('euiComboBoxPill', WAIT_FOR_EXISTS_TIME);
+        const selectedOptions = await comboBox.findAllByClassName(
+          'euiComboBoxPill',
+          WAIT_FOR_EXISTS_TIME
+        );
         if (selectedOptions.length === 0) {
           return [];
         }
-        return Promise.all(selectedOptions.map(async (optionElement) => {
-          return await optionElement.getVisibleText();
-        }));
+        return Promise.all(
+          selectedOptions.map(async optionElement => {
+            return await optionElement.getVisibleText();
+          })
+        );
       });
     }
 
-    async clear(comboBoxSelector) {
+    /**
+     * clearing value from combobox
+     *
+     * @param {string} comboBoxSelector data-test-subj selector
+     * @memberof ComboBox
```

Comment:
I have a feeling that any JSDoc parser would be able to know that this method is a member of the `ComboBox` class, do we need these lines? If not I'd prefer if you removed them.

Reply:
I use vscode extension `document this`. It autogenerated jsdoc comments(`param` and `memberof` annotations )